Automattic/jetpack
# Blogroll: Disable blog appender sites that have been added to blog roll

Fixes https://github.com/Automattic/wp-calypso/issues/82122

## Proposed changes:

Disables sites in the blog appender that have already been added to the blogroll

![CleanShot 2023-09-26 at 23 27 07@2x](https://github.com/Automattic/jetpack/assets/10482592/075f7302-ecf2-4f75-8689-f42593177267)



### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout branch
* Go to editor and add the blogroll block
* Open the blog appender and select a suggested site. It should now be added to the blogroll.
* Open the blog appender again, verify the site you just added is disabled, and that the site cannot be added to the blogroll.
